pos_pricelist/models/pos_pricelist.py

```diff
@@ -24,5 +24,6 @@ class PosPriceListConfig(models.Model):
 
     display_price_with_taxes = fields.Boolean(
         string='Price With Taxes',
-        help="Display Prices with taxes on POS"
+        help="Display Prices with taxes on POS",
+        related="iface_tax_included"
```

Comment:
should be better to detete display_price_with_taxes, and use iface_tax_included in the JS code, don't you think ?